doc/queries.md

```diff
@@ -33,6 +33,27 @@ You can filter based on the person who opened the bug.
 | `author:QUERY` | `author:descartes` matches bugs opened by `René Descartes` or `Robert Descartes` |
 |                | `author:"rené descartes"` matches bugs opened by `René Descartes`                |
 
+### Filtering by participant
+
+You can filter based on the person who participated in the bug (Opened the bug or added a comment).
```

Comment:
Could this be made `You can filter based on the person who participated in any activity related to the bug (Opened bug or added a comment`